naturalcycles/doc-tools.yml

```diff
@@ -211,16 +211,15 @@ jobs:
       - checkout
       - run:
           name: Link Jira tickets to PR
-          command: |
-            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+          command: if [ "${CIRCLE_BRANCH}" == "master" ]; then
```

Comment:
You removed the `|` part and I don't know if it still works after that. We always used `|` for multi-line commands in yaml.

Is there a reason you removed it?

How confident you are that it works without?

Reply:
I figured since it's only one bash script, that it would be treated as one line like this. I'll put it back just to be sure.